src/XeroPHP/Models/PayrollUS/Timesheet/TimesheetLine.php

```diff
@@ -103,7 +103,7 @@ public static function getProperties()
         return [
             'EarningsTypeID' => [false, self::PROPERTY_TYPE_STRING, null, false, false],
             'TrackingItemID' => [false, self::PROPERTY_TYPE_STRING, null, false, false],
-            'NumberOfUnits' => [false, self::PROPERTY_TYPE_STRING, null, false, false],
+            'NumberOfUnits' => [false, self::PROPERTY_TYPE_STRING, null, true, false],
```

Comment:
Hey, there's a brief explanation of how this works here: https://github.com/calcinai/xero-php#model-generation